docs/requirements/requirements.md

```diff
@@ -24,7 +24,7 @@ This document is in the making since we are still in the requirements engineerin
 - Researchers:
   - Researcher *Toni*
 
-- External developers *Ulm*
+- *Ulm* -- a collective of developers of a similar system
```

Comment:
I don't think that fits it too well.
They are rather doing performance / metric experiments and thus need to adapt an existing system

Reply:
How about "experienced users of similar systems; they do metric based performance experiments and regularly have to adapt the underlying reference architecture"?

Reply:
Sounds good to me